### PR TITLE
ci-bot features enhancement

### LINKED ISSUE #18: /assign only considers the last assignment if multiple assignments are done in one comment.
**What is the Issue?**  
If I try to assign multiple people to a PR by using the following:
```
/assign xyz  

/assign abc
```

Only `abc` gets assigned to the PR.

** How can the issue be reproduced?**  
1. Create a new PR.  
2. In a new comment try to assign multiple users by entering a /assign command on new lines.  
3. The bot only picks up the second assignment.  

** What is the expected output?**  
Every /assign should be considered and processed in the order of assignment.

### PR DESCRIPTION
**Pull request by pavan187**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it:**

This PR is enhacement of following features support

1. Request Reviewers from PR Body and Comment Body
  /cc @abcd
  /cc @xyz
and
  /uncc @abcd
  /uncc @xyz

2. Label support /multiple label support from PR Body or Commnet Body
   /kind xxx
   /kind yyy
and
   /remove-kind xxx
   /remove-kind yyy

3. multiple /assign support from PR body and comment Body
/assign @xxxx @yyyy
or
/assign @xxxx
/assign @yyyy input formats
and
/unassign @xxxx
/unassign @yyyy input formats

**Which issue(s) this PR **
fixes: #18

### CODE REVIEW

File: handlers/assign/assign.go [hunk 1]
```diff
@@ -14,6 +14,204 @@ const (
 	Unassign = "/unassign"
 )
 
+var (
+	assignRe = regexp.MustCompile(`(?mi)^/(un)?assign(( @?[-\w]+?)*)\s*$`)
```
Comment (m1093782566) [inline handlers/assign/assign.go]: `assignRe` -> `assignRegexp` or `AssignRegExp`?

File: handlers/label/label.go [hunk 2]
```diff
@@ -16,6 +16,123 @@ var (
 	RegRemoveLabel = regexp.MustCompile(`(?mi)^/remove-(kind|priority)\s*(.*)$`)
 )
 
+const (
+	kind = "/kind"
+	removeLable = "/remove-kind"
```
Comment (m1093782566) [inline handlers/label/label.go]: removeLable -> removeLabel?

File: handlers/assign/assign.go [hunk 3]
```diff
@@ -14,6 +14,204 @@ const (
...
+	// CCRegexp parses and validates /cc commands, also used by blunderbuss
+	CCRegexp = regexp.MustCompile(`(?mi)^/(un)?cc(( +@?[-/\w]+?)*)\s*$`)
```
Comment (m1093782566) [inline handlers/assign/assign.go]: `CCRegexp` -> `ccRegexp` ?

File: handlers/util.go [hunk 4]
```diff
@@ -22,6 +22,8 @@ var (
 
 	//assign/unassign
 	AssignOrUnassing			 = regexp.MustCompile("(?mi)^/(un)?assign(( @?[-\\w]+?)*)\\s*$")
+	// CCRegexp parses and validates /cc commands, also used by blunderbuss
+	CCRegexp 	= regexp.MustCompile(`(?mi)^/(un)?cc(( +@?[-/\w]+?)*)\s*$`)
```
Comment (m1093782566) [inline handlers/util.go]: Duplicated with the `CCRegexp` in `handlers/assign/assign.go`?

**Review by m1093782566 — APPROVED** [review]
LGTM overall except some nits.

**Comment by pavan187** [comment]
> LGTM overall except some nits.

@m1093782566 , OK thanks for the review, will fix the outstanding comments and will let you know.

File: handlers/assign/assign.go (hunk 3 above)
Comment (pavan187) [reply handlers/assign/assign.go]: > `CCRegexp` -> `ccRegexp` ?

CCRegexp var is used by other packages, if i make it lower case, other packges cannot access this variable.

Comment (m1093782566) [reply handlers/assign/assign.go]: Okay, Just keep the variable name consistent with `assignRe` or `AssignRegExp` and also consider `CCRegexp` in `handlers/util.go`.

Comment (m1093782566) [reply handlers/assign/assign.go]: So, CCRegexp -> CCRegExp ?

File: handlers/util.go (hunk 4 above)
Comment (pavan187) [reply handlers/util.go]: removed duplicates

File: handlers/assign/assign.go (hunk 3 above)
Comment (pavan187) [reply handlers/assign/assign.go]: OK

File: handlers/assign/assign.go [hunk 5]
```diff
@@ -14,6 +14,204 @@ const (
 	Unassign = "/unassign"
 )
 
+var (
+	assignRegexp = regexp.MustCompile(`(?mi)^/(un)?assign(( @?[-\w]+?)*)\s*$`)
```
Comment (m1093782566) [inline handlers/assign/assign.go]: assignRegexp -> AssignRegExp for consistency?

**Pull request merged**